Mathlib/Tactic/FinCases.lean

```diff
@@ -117,7 +122,8 @@ produces three goals with hypotheses
 
 /- TODO: In mathlib3 we ran `norm_num` when there is no `with` clause. Is this still useful? -/
 /- TODO: can we name the cases generated according to their values,
-   rather than `tail.tail.tail.head`? -/
+   rather than `tail.tail.tail.head`?
+   Update: now generates more helpful case names such as `0.2.1` -/
```

Comment:
I think you can now remove the TODO altogether, right? But maybe document the new naming behaviour in a comment somewhere.

Reply:
Thanks, changed.